content/en/contribute/code/workflow.md

```diff
@@ -251,22 +180,114 @@ The ticket needs further development.
+1. Change the Pull Request from `Draft` to `Ready for review`.
+2. The Pull Request title will be the commit message, it is important to follow the [commit message format]({{< ref "#commit-message-format" >}}) to name the Pull Request title properly. 
+3. Add a Pull Request description:
 
-Once all PRs have been approved:
+   - Add a description of changes, decisions, backstory, thinking process and any extra information to facilitate review process and reduce follow-ups. 
+   - Add videos or screenshots of the tests you did before submitting the Pull Request. This increases understanding of the work, and allows the reviewers to catch any uncovered case.
+   - Add the issue number, example: `medic/cht-core#123`
 
-1. Write a useful commit message in the PR using the [commit message format]({{< ref "#commit-message-format" >}}).
-2. Click the button to "Squash and Merge" the PR.
-3. If a backport is required cherry-pick the merged commit back to the release branches it's required in.
-4. Ensure the issue is added to the appropriate release milestone, which is the earliest semver version the change will be released in. This ensures it will be included in the release notes.
-5. Once all PRs have been merged, close the issue. This will automatically move it to "Done".
+4. Do a self-code review before asking for a review, this a good practice, almost always you will find things to fix. It saves a lot of time from you and the reviewers.
+5. Pick one reviewer for the PR and work with them until the code passes review. It is okay to include one additional reviewer that has more experience in particular subject. 
+   
+   - Coordinate with the QA engineer and add them as reviewers when you need support. For example, when major changes or new features are introduced to the codebase, security-related changes are made, and substantial user experience improvements are required.
+   
+6. Follow up on your PR, to keep momentum, the review should happen in 24h business days. If you haven't received any feedback from the reviewers after that time, check if they are available, otherwise it's okay to request a review from another person.
+7. Once the PR has been approved by a QA engineer and a developer, wait for the GitHub Actions to succeed and ensure there are no conflicts with the main branch.
+8. Double-check the [commit message format]({{< ref "#commit-message-format" >}}) is correct. Make sure to recognize collaboration in the commit description: `Co-authored-by: <gitnub user>`.
+9. Finally merge your work by selecting `Squash and merge`. This will compress all the commits into one, keeping the repository's commit history clean.
+   - If a backport is required cherry-pick the merged commit back to the release branches it's required in.
 
 ### Done
 
-Issues in this column are complete, all code has been merged into the main branch and/or release branches, and are ready for release.
+An issue is considered complete when:
+ - The documentation is finalized 
```

Comment:
Good point, I'm going to add more clarification

> Does this mean that the doc PR needs to be fixed before the issue is considered Done?

IMO, the documentation (cht-docs, readme files, or comments) has to be ready before considering the work is "done", because: 
- The team is still on that topic -- writing documentation
- We are moving to community-first, and the documentation is our way to communicate how things work